sf_stats.Api/sf_stats.Domain/Dtos/Seasons/SeasonDto.cs

```diff
@@ -14,5 +10,7 @@ public class SeasonDto
         public string DisplayName { get; set; }
         public DateTime StartDate { get; set; }
         public DateTime EndDate { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
```

Comment:
I just realized I didn't add these properties to the Dtos. Should we have a base class like DtoBase to house them?